application/handlers/event_listener_handler.py

```diff
@@ -0,0 +1,13 @@
+from constant import Status
+from application.services.event_listener import EventListener
+
+
+def listen_to_ethereum_events(event, context):
+    contract_name = event["contract_name"]
```

Comment:
As we discussed, we are going to have cloudwatch rule for this to trigger, then how do you get the contract name as an event?  correct me if I'm wrong ?

Reply:
monitor events service runs every 2 mins. This service is called from monitor event service.